src/main/resources/story/web/Listerine.story

```diff
@@ -0,0 +1,39 @@
+Meta:
+    ID_7
+
+Lifecycle:
+Examples:
+/tables/URL.table
+
+Scenario: Create new account imbd.com
+Given I am on a page with the URL '<urlListerine>'
+When I reset context
+When I click on element located `By.xpath(<faq>)`
+When I change context to element located `By.xpath(//div[@class='row header-top full'])`
+When I take screenshot
+When I hover mouse over element located `By.xpath(//input[@value='Search'])`
```

Comment:
You have this xPath `//input[@value='Search']` at least 4 times in the test. It should be a variable.